src/main/java/com/cleanroommc/groovyscript/compat/vanilla/GameRule.java

```diff
@@ -0,0 +1,54 @@
+package com.cleanroommc.groovyscript.compat.vanilla;
+
+import com.cleanroommc.groovyscript.api.GroovyBlacklist;
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.IScriptReloadable;
+import com.cleanroommc.groovyscript.api.documentation.annotations.MethodDescription;
+import com.cleanroommc.groovyscript.registry.NamedRegistry;
+import net.minecraft.world.GameRules;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class GameRule extends NamedRegistry implements IScriptReloadable {
+
+    private static final String LOG_MESSAGE = "Could not find an already existing rule with the name {}. This may be intentional! If it is, you can disable this via `gameRule.setWarnNewGameRule(false)`";
+    private final Map<String, String> defaultGameRules = new HashMap<>();
```

Comment:
fastutil map?

Reply:
uh, its object to object, i was under the understanding that either `Object2ObjectOpenHashMap` or `HashMap` are fine, and theres little difference. is this incorrect?

Reply:
fastutil is always the better choice for hash maps